contrib/podman-next/fcos-podmanimage/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/fedora/fedora-coreos:stable
+
+# Setup the podman-next copr repo
+# Note: This might need to be updated from time to time to the latest GPG key.
+# The repo config itself is release agnotic but the GPG key might change.
```

Comment:
Is there a way to grab the key from the source (somewhere)?  IIRC the `ADD` instruction can reference something stored in a git repo.

Reply:
yes, there is. Great idea. I'll update this and work on other suggestions. Hold off any further review until then. Thanks!